exercises/concept/vehicle-purchase/test-vehicle-purchase.bats

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bats
+load bats-extra
+
+@test requires_a_license_for_a_car {
+    ## task 1
+    run gawk -F, -f vehicle-purchase.awk <<< 'needs_license,car'
+    assert_success
+    assert_output "true"
+}
+
+@test requires_a_license_for_a_truck {
+    ## task 1
+    run gawk -F, -f vehicle-purchase.awk <<< 'needs_license,ice cream truck'
+    assert_success
+    assert_output "true"
+}
+
+@test does_not_require_a_license_for_a_bike {
+    ## task 1
+    run gawk -F, -f vehicle-purchase.awk <<< 'needs_license,bike'
+    assert_success
+    assert_output ""
+}
+
+@test does_not_require_a_license_for_a_stroller {
+    ## task 1
+    run gawk -F, -f vehicle-purchase.awk <<< 'needs_license,stroller'
+    assert_success
+    assert_output ""
+}
+
+@test requires_a_license_for_a_different_car {
+    ## task 1
+    run gawk -F, -f vehicle-purchase.awk <<< 'needs_license,streetcar'
+    assert_success
+    assert_output "true"
+}
+
+
+@test price_is_reduced_to_80%_for_age_below_3 {
+    ## task 2
+    run gawk -F, -f vehicle-purchase.awk << END_INPUT
+resell_price,40000,2
```

Comment:
All the tests other than this one have exactly one line of input. Can we make them all one line (and maybe use heredocs, too)?

Reply:
Yes, herestring :)